pkg/sql/codegen/tensorflow/template_explain.go

```diff
@@ -79,5 +84,11 @@ explain.explain(datasource="{{.DataSource}}",
                 feature_metas=feature_metas,
                 label_meta=label_meta,
                 model_params=model_params,
-                save="{{.Save}}", **summaryAttrs)
+                save="{{.Save}}",
+                result_table="{{.ResultTable}}",
+                hdfs_namenode_addr="{{.HDFSNameNodeAddr}}",
+                hive_location="{{.HiveLocation}}",
+                hdfs_user="{{.HDFSUser}}",
+                hdfs_pass="{{.HDFSPass}}",
```

Comment:
Is there a way to avoid passing `hdfs_` in all subsequent calls? Is it a good idea to wrap `hive/hdfs*` in a single struct?

Reply:
Will try to refactor this in incoming PRs, also need to be done in `predict.py`